stores/metadata_test.go

```diff
@@ -2465,6 +2465,13 @@ func TestRenameObjects(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// Wait for prune loop to run.
+	ts := time.Now()
+	ss.triggerSlabPruning()
```

Comment:
Does it work without this as well by just waiting for the prune loop after every rename? Because if we manually trigger it here we no longer confirm that `RenameObject` itself also triggers slab pruning.

Reply:
Yeah making every rename blocking works too I think. The test is a lot slower though (~ x8). I didn't see the big deal but you are right, test is less valuable with the manual trigger at the end.